assets/js/show-task.js

```diff
@@ -1,6 +1,7 @@
+import deleteTaskEvent from "/assets/js/delete-task.js";
+import { HIDE_CLASS, icons } from "/assets/js/constants.js";
 import createButton from "/assets/js/factory/createButton.js";
-import { icons } from "/assets/js/constants.js";
-function showTask(inputValue, formattedDate, taskList, form) {
+function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
```

Comment:
use one extra line break between the imports and where the code begins.

```suggestion

function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
```